main.go

```diff
@@ -54,6 +56,44 @@ func sources() []string {
 }
 
 func main() {
+	var (
+		cfgFile        string
+		debug, version bool
+	)
+	flag.StringVar(&cfgFile, "c", "policies.yaml", "Path to configuration file")
+	flag.BoolVar(&version, "v", false, "Print version")
+	flag.BoolVar(&version, "D", false, "Set debug")
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "USAGE: doorman -c policies.yaml\n")
+		fmt.Fprintf(os.Stderr, "\nCLI options\n-----------\n\n")
+		fmt.Fprintf(os.Stderr, "  -c: string\n\tPath to the configuration file (default `policies.yaml`)\n")
+		fmt.Fprintf(os.Stderr, "  -D:\n\tSet log level to `debug`\n")
+		fmt.Fprintf(os.Stderr, "  -v:\n\tDisplay the current server version\n")
+		fmt.Fprintf(os.Stderr, "  -h:\n\tDisplay this help\n")
+		fmt.Fprintf(os.Stderr, "\n\nEnvironment variables\n---------------------\n\n")
+		fmt.Fprintf(os.Stderr, "  POLICIES: string\n\tPath to the configuration file\n\n")
+		fmt.Fprintf(os.Stderr, "  VERSION_FILE: string\n\tPath to the version file served at /__version__\n\n")
+		fmt.Fprintf(os.Stderr, "  GITHUB_TOKEN: string\n\tGithub API token used to reload the config.\n\n")
+		fmt.Fprintf(os.Stderr, "  LOG_LEVEL: string\n\tServer log level (one of: fatal, error, warn, debug)\n\n")
```

Comment:
This gives me the feeling that the CLI parameters are useless (apart from `--help` and `--version`)